archspec/cli.py

```diff
@@ -18,7 +19,36 @@ def main():
     """archspec command line interface"""
 
 
+def cb_cpu_dag(ctx, param, value):
+    """Print Direct Acyclic Graph (DAG) for known CPU microarchitectures."""
+
+    def node_label(uarch):
+        """Create node label for specified Microarchitecture instance."""
+        res = uarch.name
+        if uarch.parents and uarch.vendor != uarch.parents[0].vendor:
+            res += " (" + uarch.vendor + ")"
+        return res
+
+    cpu_uarch_dag = Digraph()
+
+    for key in archspec.cpu.TARGETS:
+        uarch = archspec.cpu.TARGETS[key]
```

Comment:
Can this be written as:
```suggestion
    for uarch in archspec.cpu.TARGETS.values():
```
?